linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -687,6 +688,10 @@ impl ClientWrapper {
         WalletState::from_file(self.wallet_path().as_path())
     }
 
+    pub fn wallet(&self) -> Result<Wallet> {
```

Comment:
Optional and minor: I recall seeing somewhere that the `get_` prefix is useful as a marker that the operation can be expensive (e.g., reading from a file). Based on the same motivation, would it make sense to use a different name here, like `load_wallet`?

Reply:
Right, good point; I thought of that, too, and forgot again. :sweat_smile: 

How about `read_wallet`? I think that makes it clear that it's being read from disk.